processor.go

```diff
@@ -50,11 +50,21 @@ func (dp *DefaultProcessor) execute(ctx context.Context, notifier Notifier, stat
 	}
 	url := "gs://" + bucket + "/" + name
 
+	service := &WatchService{ctx}
+	topic, err := service.topicFor(url)
+	if err != nil {
+		return err
+	}
+	if topic == "" {
+		log.Infof(ctx, "No topic found for %q", url)
+		return nil
```

Comment:
At first I saw `return nil`, I thought it should return an error,
but I understand that it should not return an error to prevent OCN retry loop, is this correct?
(ref: https://cloud.google.com/storage/docs/object-change-notification#_Reliable_Delivery)